nursery/wmi-get-antivirus.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: wmi get antivirus
```

Comment:
Please update this name to reflect capa's [rule name convention described here](https://github.com/mandiant/capa-rules/blob/master/doc/format.md#rule-name).